setting-up-ocarina.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: docpost
+title: Setting up Ocarina and OAuth
+date_published: 
+date_modified:  2021-07-05 15:01:00 +0000
+author: BioWilko
+maintainer: BioWilko
+---
+
+### 0. Register for an account via Majora
+[Read the instructions for how to register](register)
+
+### 1. Acquire permission to access a data view
+A data view is basically a list of metadata fields. If you are granted access to a data view, you can see all of the fields covered by that data view.
+Every data view is associated with a short name, which we sometimes call a `code` or Majora data view code (`mdv` code).
+
+Guidance on how to obtain access to a view is in progress, check back after the consortium agreement has been distributed.
+This document has been written ahead of time and we are not currently granting permissions to any restricted views. Do not attempt to contact anyone to arrange access at this time.
+
+### 2. Install the Ocarina client
+
+[Ocarina](https://github.com/SamStudio8/ocarina/tree/master/ocarina) is a command line tool that is used to connect to Majora and perform actions with elevated privileges that are not possible on the website.
+It is not terribly difficult to use, but ideally you will have used a command line tool before.
+
+You can install the latest version with the Python package manager. You'll probably want to install it into a conda environment on the shared node.
+
+```
+conda create -n mdv-ocarina python=3.7
+conda activate mdv-ocarina
+pip install git+https://github.com/samstudio8/ocarina.git
```

Comment:
Perhaps an instruction to remind users to periodically check the ocarina changelog https://github.com/SamStudio8/ocarina/blob/master/CHANGELOG.md and update their client